features/admin.copilot.v1/components/copilot-chat.scss

```
@@ -0,0 +1,517 @@
+                    }
+                    
+                    .copilot-message-bubble {
+                        padding: $copilot-spacing-md;
+                        background-color: transparent !important;
+                        color: $copilot-text-color-primary;
+                        border-radius: 0 !important;
+                        font-size: $copilot-font-size-md;
+                        line-height: $copilot-line-height-normal;
+                        overflow-wrap: break-word;
+                        border: none !important;
+                        // Optimize rendering performance during streaming
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)